doc/tutorial/image/tutorial-basic-drawings.doc

```diff
@@ -0,0 +1,112 @@
+
+/**
+  \page tutorial-basic-drawings Tutorial: How to draw basic drawings
+  \tableofcontents
+
+\section image_display_intro Introduction
+
+In this tutorial you will learn how to draw basic drawings with ViSP either on Unix-like systems (including OSX, Fedora, Ubuntu, Debian, ...) or on Windows.
+
+Note that all the material (source code and images) described in this tutorial is part of ViSP source code and could be downloaded using the following command:
+
+\code
+$ svn export https://github.com/lagadic/visp.git/trunk/tutorial/image
+\endcode
+
+\section image_display_default How to display an image
+
+ViSP <a href="https://visp.inria.fr/gui/">gui module</a> provides Graphical User Interfaces capabilities. This module may use several optional third-party libraries which are: <a href="http://visp.inria.fr/3rdparty_gui/">OpenCV, X11, GTK, GDI, Direct3D</a>. We recommand to use X11 on unix-like systems thanks to vpDisplayX class and GDI on Windows thanks to vpDisplayGDI. 
+
+The following example also available in tutorial-image-display.cpp shows how to create a gray level 3840x2160 image with all the pixels set to 128, and display a red circle with 200 pixel radius in the middle of the image. 
+
```

Comment:
Line 20 is wrong. You are explaining below tutorial-viewer.cpp not tutorial-image-display.cpp